specification/loadtestservice/LoadTestService/client.tsp

```diff
@@ -24,7 +24,7 @@ interface AdministrationOperations {
   getTestFile is LoadTestAdministration.getTestFile;
   listTestFiles is LoadTestAdministration.listTestFiles;
   listTests is LoadTestAdministration.listTests;
-  #suppress "@azure-tools/typespec-azure-core/byos"
+  #suppress "@azure-tools/typespec-azure-core/byos" //It is limiting for users to have storage first before creating a load test also it will be breaking change for us.
```

Comment:
```suggestion
  #suppress "@azure-tools/typespec-azure-core/byos" "It is limiting for users to have storage first before creating a load test also it will be breaking change for us."
```

suppress takes a 2nd argument which is the message no need to use comment